src/nats.h

```diff
@@ -1281,26 +1281,39 @@ typedef struct jsOptions
 
         struct jsOptionsSubscribePullAsync
         {
-                // Options to control automatic Fetch flow control.
-                //
-                // The number of messages to ask for in a single request, and if
-                // we should try to fetch ahead, KeepAhead more than we need to
-                // finish the current request. Fetch this many messages ahead of
-                // time.
-                int FetchSize;
-                int KeepAhead;
+                // Lifetime of the subscription (completes when any one of the
+                // targets is reached).
+                int64_t                 TimeoutMillis;
```

Comment:
nit: I know that it may sound obvious that `TimeoutMillis` means timeout in milliseconds, but I would have used `Timeout` but document above that the time is in milliseconds.

Reply:
👍 I went a little overboard after confusing myself.